jaxrs/src/main/resources/kafka/producer.props

```diff
@@ -0,0 +1,9 @@
+bootstrap.servers=172.19.0.3:9092
```

Comment:
As said, for now this is fine.

However, version 0.0.5 of the kafka-cdi-extension is able to resolve simple expressions, like:
```
#{HOST}:#{PORT}"
```

Perhaps its a good idea to look at `javax.el` APIs for this. See AGPUSH-2104